src/Resources/views/CRUD/base_array_macro.html.twig

```diff
@@ -8,17 +8,42 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% macro render_array(value, inline) %}
-    {% from _self import render_array %}
-    {% for key, val in value %}
-        {% if val is iterable %}
-            [{{ key }} => {{ render_array(val, inline) }}]
-        {% else %}
-            [{{ key }} => {{ val }}]
```

Comment:
WDYT about improving the inline rendering to, from
```
[key => value][key => value][key => value]
```
to
```
[key => value, key => value, key => value]
```

?

Reply:
It makes sense and looks much user-friendly 🙂

Reply:
Should we also use non-breakable space arround the `=>` ?